README.md

```diff
@@ -58,6 +58,12 @@
    pip3 install -e '.[full]'
    ```
 
+   For the PyTorch version of the ogbg workload, you might have to install
```

Comment:
nit: is it actually required to install? if so can we change "you might have to" to "you need to"